frankfurt/nucleus-dev/nodeport-svc.yaml

```diff
@@ -1,18 +1,24 @@
 apiVersion: v1
 kind: Service
 metadata:
+  creationTimestamp: null
   labels:
     app: nucleus-dev
-  name: nucleus-dev-elb
+  name: nucleus-nodeport
   namespace: nucleus-dev
 spec:
+  externalTrafficPolicy: Cluster
   ports:
-  - name: http
-    port: 80
+  - name: https
```

Comment:
in the basket equivalent it's using http and port 80 still. Is that the port at which it communicates with the service?

Reply:
our nodeports are typically unencrypted, I don't fully understand why this is https/443